include/ofi_util.h

```diff
@@ -256,6 +256,8 @@ struct util_ep {
 	ofi_ep_progress_func	progress;
 	struct util_cmap	*cmap;
 	fastlock_t		lock;
+	ofi_fastlock_acquire_t	ep_lock_acquire;
+	ofi_fastlock_release_t	ep_lock_release;
```

Comment:
I'm reading the commits backwards -- please merge this commit with the next one.